packages/ui/src/ui/common/thor/ypath.js

```diff
@@ -79,9 +79,17 @@ thorYPath.getBoolean = function (node, path) {
  * @returns {number|*}
  */
 thorYPath.getNumber = function (node, path, defaultValue) {
-    const value = thorYPath.get(node, path);
-
-    return convertToNumber(value, defaultValue);
+    try {
+        const value = thorYPath.get(node, path);
+        return convertToNumber(value, defaultValue);
+    } catch (e) {
+        const curError = {
+            message: `thorYPath.getNumber: failed to convert field with path: "${path}".`,
+        };
+        if (e?.inner_errors) e.inner_errors.push(curError);
```

Comment:
The action of adding an item to `inner_errors` should be exception safe. I think it is a bad idea to rely on that type of inner_errors might be defined only as `Array` or `undefined`. It is better to check it and handle it properly, probably, we need helper function.

Reply:
i have added a helper function `appendInnerErrors`:

you could use this compare link to watch recent chages: https://github.com/ytsaurus/ytsaurus-ui/compare/4e9c258381505e3e485be3ab3df56c35565dc8fb..7852283a95e27c204df8bde4358e56f8a80ed474